src/test/java/edu/ucsb/cs156/happiercows/services/SystemInfoServiceImplTests.java

```diff
@@ -28,7 +28,7 @@ class SystemInfoServiceImplTests  {
   void test_getSystemInfo() {
     SystemInfo si = systemInfoService.getSystemInfo();
     assertTrue(si.getSpringH2ConsoleEnabled());
-    assertTrue(si.getShowSwaggerUILink());
+    assertTrue(!si.getShowSwaggerUILink());
```

Comment:
```suggestion
    assertFalse(si.getShowSwaggerUILink());
```

This might be more clear.

Reply:
You'll need an import for it but that's straightforward.

Reply:
It's just:
```java
import static org.junit.jupiter.api.Assertions.assertFalse;
```